build.gradle

```diff
@@ -71,7 +71,7 @@ def elkSources = [
   "${elkRepo}/plugins/org.eclipse.elk.graph.json/src",
   "${elkRepo}/plugins/org.eclipse.elk.alg.common/src",
   // The algorithms
-  "${elkRepo}/plugins/org.eclipse.elk.alg.disco/src",
+  //"${elkRepo}/plugins/org.eclipse.elk.alg.disco/src",
```

Comment:
Maybe add a comment why this is not included here and maybe a FIXME to investifate the problem.